src/components/Leaderboard.js

```diff
@@ -77,9 +81,9 @@ const Leaderboard = ({ content, limit = 100 }) => (
           emoji = ":3rd_place_medal:"
         }
         return (
-          <Item key={idx} to={`${githubUrl}${username}`}>
+          <Item key={idx} to={hasGitHub ? `${githubUrl}${username}` : "#"}>
             <ItemNumber>{idx + 1}</ItemNumber>
-            <Avatar src={`${githubUrl}${username}.png?size=40`} />
+            <Avatar src={avatarImg} alt={`${username} GitHub avatar image`} />
```

Comment:
Screenreaders already declare it is an image

```suggestion
            <Avatar src={avatarImg} alt={`${username} GitHub avatar`} />
```